var/spack/repos/builtin/packages/py-nvidia-dali/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import platform
+
+from spack.package import *
+
+
+class PyNvidiaDali(PythonPackage):
+    """A GPU-accelerated library containing highly optimized building blocks and
+    an execution engine for data processing to accelerate deep learning training
+    and inference applications. """
+
+    homepage = "https://developer.nvidia.com/dali"
+    url = "https://developer.download.nvidia.com/compute/redist/"
+
+    maintainers = ["thomas-bouvier"]
+
+    arch = platform.machine()
```

Comment:
We should also add a condition so that these are only applied on linux, not macos/windows